### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by Cyquen**

### CODE REVIEW

File: data/shelters/Shelters in Free State - DOJ 2012 database.csv [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+2,Philani ,Xhariep ,NPO ,7,Ms Prudence Ramolefe,051 409 0566/2,071 857 4086
+3,Tshireletso ,Xhariep ,NPO ,12,Ms Prudence Ramolefe,051 409 0566/2,071 857 4086
+4,Siville ,Xhariep ,NPO ,6,Ms Prudence Ramolefe,051 409 0566/2,071 857 4086
+5,Serobe (The Nest shelter),Motheo ,NPO ,16,Ms Prudence Ramolefe,051 409 0566/2,071 857 4086
+6,Thusanang Advice Centre,Thabo Mofutsanyane ,NPO ,8,Ms Prudence Ramolefe,051 409 0566/2,071 857 4086
+7,Child and Family Welfare,Thabo Mofutsanyane ,,6,Ms Prudence Ramolefe,051 409 0566/2,071 857 4086
+8,Tumahole ,Fezile Dabi ,NPO ,10,Ms Prudence Ramolefe,051 409 0566/2,071 857 4086
+9,Goldfields Family Advice Organization,Lejweleputswa ,NPO ,16,Ms Prudence Ramolefe,051 409 0566/2,071 857 4086
+,,,,92,,,
+,,,,,,,
```
Comment (schalkventer) [inline data/shelters/Shelters in Free State - DOJ 2012 database.csv]: @Cyquen Why are these rows empy?

Comment (Cyquen) [reply data/shelters/Shelters in Free State - DOJ 2012 database.csv]: I was adding how much capacity there is for all shelters (e.g. the number of beds for Free State is 92). Must that row come out?

Comment (schalkventer) [reply data/shelters/Shelters in Free State - DOJ 2012 database.csv]: Ok, I see. However this specific file has another `,,,,,,,` row below that?

Comment (Cyquen) [reply data/shelters/Shelters in Free State - DOJ 2012 database.csv]: The next row is blank, I am not sure what I did that it thought it is a row, but there's no data in that row and it is correct that there shouldn't be any data there - there are 9 shelters in free state

**Pull request merged**